src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
It is a best practice to have enum names in singular form, consider renaming the enum to 'Color' instead of 'Colors'.